pre_commit/languages/docker.py

```diff
@@ -84,16 +85,19 @@ def get_docker_user() -> str:  # pragma: win32 no cover
 
 
 def docker_cmd() -> Tuple[str, ...]:  # pragma: win32 no cover
-    return (
+    cmds = [
         'docker', 'run',
         '--rm',
-        '-u', get_docker_user(),
```

Comment:
let's instead change `get_docker_user` to return `Tuple[str, ...]` and change this line to be `*get_docker_user(),` and in the case where we can't retrieved uid/gid return `()`

then you don't need to change the rest of the code

Reply:
This is very neat! Will adopt in latest commit.